include/splinterdb/splinterdb.h

```diff
@@ -67,6 +66,9 @@ typedef struct {
    uint64 max_branches_per_node;
    uint64 use_stats;
    uint64 reclaim_threshold;
+   uint64 num_memtable_bg_threads;
```

Comment:
Give some comments / guidance explaining how these config params should be set.

Reply:
Memtable bg-threads are latency sensitive. Normal bg-threads are driven for compaction, so I/O centric.

Reply:
A couple small suggestions
- Note that it is an error to for one to be zero while the other is non-zero
- rather than "affect" latency of inserts, additional memtable background threads can "improve" insert latencies, or maybe more specifically "reduce insert tail latencies."
- Not sure if this really matters to app devs, but "normal" background threads work on compacting branches _and building filters_

Reply:
Wordsmith'ed this a bit more.

Reply:
Reworked this suggested rewrite to fold it into what was there previously. Uploaded in latest commit coming up ...